.github/workflows/p2p-version.yaml

```diff
@@ -6,6 +6,9 @@ on:
       version:
         description: "The next semantic version"
         value: ${{ jobs.increment-version.outputs.version }}
+      previous_version:
```

Comment:
what did we need this for?

Reply:
if we want to have 2 pipelines with the same tag, we should have a separate workflow just for managing the version.
so what I'm thinking of doing is having a "versioning" workflow that will just create a new tag, and the other 2 are triggered workflows by a tag creation. This previous_version is used to get the version that the "versioning" pipeline created. We could use the github tag value but that would come with the prefix and we'd need to "manually" remove it, this way is cleaner